_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -29,6 +29,11 @@ Delivery pipeline for your infrastructure code. This guide will walk you through
 pipeline for your favorite infrastructure as code tools (e.g., Terraform) using your favorite CI/CD platform (e.g.,
 Jenkins, Circle, GitLab, etc).
 
+Here is what the pipeline looks like in action:
+
+image::/assets/img/guides/infrastructure-cicd-pipeline/walkthrough.gif[]
```

Comment:
NIT: consider adding a sentence to tell the user there's a longer, more detailed video walkthrough below, plus the guide will expose all the details

Reply:
I decided to link to the youtube video in the caption just like with the blog post and remove it from the bottom.